Update issue templates (#89)
stackrox/kube-linter: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
title: "[FEATURE_REQUEST]"
labels: ''
assignees: ''

---

**Description of the problem/feature request**

> A clear and concise description of what the problem is, or the what the new feature is targeting at

Reviewer: ```suggestion
A clear and concise description of the problem, or the proposed new feature.
```

**Description of the existing behavior vs. expected behavior**

> If possible, please paste in the existing KubeLinter output along with the input used, and point out which part should be modified (expected output).

Reviewer: ```suggestion
If applicable, please paste in the existing KubeLinter output along with the input used, and point out which part should be modified (expected output).
```
